custom_components/sat/__init__.py

```
@@ -17,11 +17,16 @@
     CLIMATE,
     SENTRY,
     COORDINATOR,
+    OPTIONS_DEFAULTS,
     CONF_MODE,
     CONF_DEVICE,
-    CONF_ERROR_MONITORING, OPTIONS_DEFAULTS,
+    CONF_ERROR_MONITORING,
+    SERVICE_RESET_INTEGRAL,
```

Ruff (0.8.2)

`.const.SERVICE_RESET_INTEGRAL` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)